src/main/java/com/sollace/coppergolem/client/CopperGolemEntityRenderer.java

```diff
@@ -1,16 +1,16 @@
 package com.sollace.coppergolem.client;
 
 import net.minecraft.block.Oxidizable.OxidationLevel;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.render.VertexConsumerProvider;
 import net.minecraft.client.render.entity.EntityRendererFactory.Context;
+import net.minecraft.client.render.entity.MobEntityRenderer;
 import net.minecraft.client.render.entity.feature.FeatureRenderer;
 import net.minecraft.client.render.entity.feature.FeatureRendererContext;
 import net.minecraft.client.render.model.json.ModelTransformation;
 import net.minecraft.client.util.math.MatrixStack;
 import net.minecraft.entity.EquipmentSlot;
 import net.minecraft.item.ItemStack;
-import net.minecraft.client.MinecraftClient;
-import net.minecraft.client.render.VertexConsumerProvider;
-import net.minecraft.client.render.entity.MobEntityRenderer;
```

Comment:
I wasn't sure if this group being out-of-order matters, but it's easy enough for me to revert if it does.